style.css

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
lots of unnecessary line breaks and incorrect formatting in this document, please fix all of it